File: Lab3/src/main/java/org/example/domain/Autocar.java
package org.example.domain;

public class Autocar extends Entity<Long>{

    private String nrInmatriculare;
    private int nrLocuri;

    public Autocar(String nrInmatriculare, int nrLocuri) {
        this.nrInmatriculare = nrInmatriculare;
        this.nrLocuri = nrLocuri;
    }

    private String getNrInmatriculare() {
        return nrInmatriculare;
    }
    private void setNrInmatriculare(String nrInmatriculare) {
        this.nrInmatriculare = nrInmatriculare;
    }
    private int getNrLocuri() {
        return nrLocuri;
    }
    private void setNrLocuri(int nrLocuri) {
        this.nrLocuri = nrLocuri;
    }

    @Override
    public String toString() {
        return "Autocar{" +
                "nrInmatriculare='" + nrInmatriculare + '\'' +
                ", nrLocuri=" + nrLocuri +
                '}';
    }
}

File: Lab3_c_sharp/domain/Autocar.cs
namespace Lab3_c_sharp.domain;


public class Autocar : Entity
{
    private string numarInmatriculare;
    private int numarTotalLocuri;

    public Autocar(int id, string numarInmatriculare, int numarTotalLocuri) : base(id)
    {
        this.numarInmatriculare = numarInmatriculare;
        this.numarTotalLocuri = numarTotalLocuri;
    }
    
    public string NumarInmatriculare
    {
        get => numarInmatriculare;
        set => numarInmatriculare = value;
    }
    public int NumarTotalLocuri
    {
        get => numarTotalLocuri;
        set => numarTotalLocuri = value;
    }

}